application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java

```diff
@@ -83,7 +84,13 @@ public HelpSystemHelper(JDA jda, Config config, Database database) {
         categoryRoleSuffix = helpConfig.getCategoryRoleSuffix();
     }
 
-    RestAction<Message> sendExplanationMessage(MessageChannel threadChannel) {
+    RestAction<Message> sendExplanationMessage(GuildMessageChannel threadChannel) {
+        return MessageUtils.mentionSlashCommand(threadChannel.getGuild(), CloseCommand.COMMAND_NAME)
+            .flatMap(command -> sendExplanationMessage(threadChannel, command));
+    }
+
+    private RestAction<Message> sendExplanationMessage(GuildMessageChannel threadChannel,
+            String command) {
```

Comment:
the name `command` is confusing. what command? just call it `closeCommandMention`. then its clear what it is. and also what its purpose is.